recipe/meta.yaml

```diff
@@ -253,7 +270,7 @@ outputs:
         # decide on the cuda variant
         # https://github.com/conda-forge/tensorflow-feedstock/issues/162
         - {{ compiler('cuda') }}                 # [cuda_compiler_version != "None"]
-        - bazel
+        - bazel <5
```

Comment:
Above we have `bazel 4.2.1`, here `bazel <5`; Is there a reason for the difference?